packages/launch/src/focused-launch/summary/focused-launch-summary-item/index.tsx

```diff
@@ -30,19 +30,22 @@ export const LeadingContentSide: React.FunctionComponent< LeadingSideProps > = (
 };
 
 export const TrailingContentSide: React.FunctionComponent< TrailingSideProps > = ( props ) => {
-	if ( props.price ) {
-		return <div className="focused-launch-summary-item__price-cost">{ props.price }</div>;
+	if ( props.warningNote ) {
+		return <div className="focused-launch-summary-item__warning-note">{ props.warningNote }</div>;
 	}
-	return <div className="focused-launch-summary-item__warning-note">{ props.warningNote }</div>;
+	return <div className="focused-launch-summary-item__price-cost">{ props.price }</div>;
 };
```

Comment:
For more clarity and future-proofing, we should probably refactor `TrailingSideProps` to be something like

```
{
  node: ReactNode;
  noteType: 'PRICE' | 'WARNING';
}
```

and refactor the component's code to something like

```
return <div
  className={classNames({
    'focused-launch-summary-item__warning-note': noteType === 'WARNING',
    'focused-launch-summary-item__price-cost': noteType === 'PRICE',
  })}
>
  { props.node }
</div>;
```

Reply:
Love this.